packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -48,6 +51,7 @@ const defaultProps = {
   isDisabled: false,
   variant: 'neutral',
 };
+const WrappedButtonBase = onClickOutside(DropdownButtonBase);
```

Comment:
I think we can do this wrapping in _DropdownButtonBase instead of here like below 
`export default reactOnClickOutside(DropdownButtonBase);`